tests/Behat/ElasticsearchContext.php

```diff
@@ -88,15 +89,17 @@ private function deleteIndexes(): void
         $finder = new Finder();
         $finder->files()->in($this->elasticsearchMappingsPath)->name('*.json');
 
-        $indexClient = $this->client->indices();
+        $indexes = [];
 
         foreach ($finder as $file) {
-            $index = $file->getBasename('.json');
-            if (!$indexClient->exists(['index' => $index])) {
-                continue;
-            }
+            $indexes[] = $file->getBasename('.json');
+        }
 
-            $indexClient->delete(['index' => $index]);
+        if ($indexes !== []) {
+            $this->client->indices()->delete([
```

Comment:
remove all indexes within a single http request instead of multiple HTTP requests